sql/moz-fx-data-shared-prod/mozilla_vpn_derived/waitlist_funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH send_ids AS (
+  SELECT
+    SendID,
+    ARRAY_AGG(DISTINCT URL) AS URLs,
+  FROM
+    `mozilla-cdp-prod`.sfmc.clicks
+  WHERE
+    URL LIKE "%://vpn.mozilla.org%"
+  GROUP BY
+    SendID
+),
+jobs AS (
+  SELECT
+    SendID,
+    SentTime,
+    URLs,
+  FROM
+    `mozilla-cdp-prod`.sfmc.sendjobs
+  INNER JOIN
+    send_ids
+  USING
+    (SendID)
+)
+SELECT
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.sent
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS sent,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.bounces
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS bounced,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.opens
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS opened,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.unsubs
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
+  ) AS unsubscribed,
+  (
+    SELECT
+      COUNT(DISTINCT EmailAddress),
+    FROM
+      `mozilla-cdp-prod`.sfmc.clicks
+    JOIN
+      jobs
+    USING
+      (SendID)
+    WHERE
+      TIMESTAMP_DIFF(EventDate, SentTime, DAY) < 14
```

Comment:
I was wrong about there only being one send job so far, see slack for more info.

I limited to 14 days because I did `APPROX_QUANTILE(TIMESTAMP_DIFF(EventDate, SentTime, DAY), 20)[OFFSET(19)]` and got a 95th percentile of 10 days for clicks, and then rounded that up to the nearest week to avoid weekday variation.